### PR TITLE
structured data in json-ld : improvements

### PR DESCRIPTION
**Pull request by jf-viguier**
- Add sku to product
- If no brand available : uses shop name
- Add a general identifier in product and offer : gtin13 or isbn or mpn (product reference)
- Bug fix priceValidUntil

### CODE REVIEW

File: templates/_partials/microdata-jsonld.tpl [hunk 1]
```diff
@@ -96,10 +96,19 @@
...
+    {if $product.reference}"sku": "{$product.reference}",{/if}
+    {if $product.ean13}
+    "gtin13": "{$product.ean13}",
+    {else if $product.upc}
+    "gtin13": "0{$product.upc}",
+    {else if $product.isbn}
+    "isbn": "{$product.isbn}",
+    {else if $product.reference}
+    "mpn": "{$product.reference}",
+    {/if}
```
Comment (Progi1984) [inline templates/_partials/microdata-jsonld.tpl]: Please indent code :)
```suggestion
    {if $product.ean13}
      "gtin13": "{$product.ean13}",
    {else if $product.upc}
      "gtin13": "0{$product.upc}",
    {else if $product.isbn}
      "isbn": "{$product.isbn}",
    {else if $product.reference}
      "mpn": "{$product.reference}",
    {/if}    
```

File: templates/_partials/microdata-jsonld.tpl [hunk 2]
```diff
@@ -133,6 +142,10 @@
         "gtin13": "{$product.ean13}",
         {else if $product.upc}
         "gtin13": "0{$product.upc}",
+        {else if $product.isbn}
+        "isbn": "{$product.isbn}",
+        {else if $product.reference}
+        "mpn": "{$product.reference}",
```
Comment (Progi1984) [inline templates/_partials/microdata-jsonld.tpl]:
```suggestion
        {else if $product.isbn}
          "isbn": "{$product.isbn}",
        {else if $product.reference}
          "mpn": "{$product.reference}",
```

File: templates/_partials/microdata-jsonld.tpl (hunk 1 above)
Comment (NeOMakinG) [reply templates/_partials/microdata-jsonld.tpl]: That will looks weird client-slide.. that's why I gave up on this

Comment (Progi1984) [reply templates/_partials/microdata-jsonld.tpl]: It's JSON Ld... only robots look at this ;)

**Pull request merged**